### PR TITLE
add balancesheet

### PR DESCRIPTION
**Pull request by ulrikaakre**

### CODE REVIEW

File: airbyte-integrations/connectors/source-tripletex-api/source_tripletex_api/source.py [hunk 1]
```diff
@@ -126,6 +126,17 @@ def path(
     ) -> str:
         return "ledger/account"
 
+class BalanceSheet(TripletexApiStream):
+    primary_key = "id"
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        return None
```
Comment (RudyiVT) [inline airbyte-integrations/connectors/source-tripletex-api/source_tripletex_api/source.py:133]: The balance endpoint requires `dateFrom` and `dateTo` parameters, `next_page_token ` shouldn't return None.
Try to use the `Postings` stream as a template.

File: airbyte-integrations/connectors/source-tripletex-api/source_tripletex_api/source.py [hunk 2]
```diff
@@ -126,6 +126,17 @@ def path(
...
+
+    def path(
+            self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> str:
+        return "balanceSheet"
+
 
 class SourceTripletexApi(AbstractSource):
```
Comment (RudyiVT) [inline airbyte-integrations/connectors/source-tripletex-api/source_tripletex_api/source.py:141]: You need to register a new stream by adding a corresponding class to the return statement. 
Example: `[Postings(config=config), Departments(config=config), Accounts(config=config), BalanceSheet(config=config)]`

**Review by RudyiVT — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-tripletex-api/source_tripletex_api/source.py (hunk 1 above)
Comment (ulrikaakre) [reply airbyte-integrations/connectors/source-tripletex-api/source_tripletex_api/source.py:133]: As both Postings and Balance use dateFrom and DateTo could the PostingStream be renamed and use for both or do they have to be unique @RudyiVT?

Comment (RudyiVT) [reply airbyte-integrations/connectors/source-tripletex-api/source_tripletex_api/source.py]: Shouldn't be any conflicts, we need to test it

**Review by RudyiVT — APPROVED** [review]

**Pull request merged**